lib/MusicBrainz/Server/Entity/URL/BBCMusic.pm

```diff
@@ -1,26 +0,0 @@
-package MusicBrainz::Server::Entity::URL::BBCMusic;
-
-use Moose;
-
-extends 'MusicBrainz::Server::Entity::URL';
-with 'MusicBrainz::Server::Entity::URL::Sidebar';
-
-override href_url => sub {
-    shift->url->as_string =~ s{^http:}{https:}r;
-};
```

Comment:
For the record, all BBC Music URLs in MusicBrainz Database are `https` now. This override was obsolete already.

Reply:
We should probably check whether an override exists (and get rid of it) when we bot- or cron- change URLs to https :)